contributed_definitions/NXem.nxdl.xml

```diff
@@ -171,145 +236,183 @@
          nested groups, fields and attributes. An application definition can offer a
-         place to hold these nested representations; however at the cost of generality.
+         place to hold these nested representations; however as discussed
+         at the cost of generality.
          
          Which specific details matter for answering scientific research questions is
          a difficult question to answer by a single team of scientists, especially
          if the application definition is to speak for a number of vendors. What makes
-         it especially challenging if the application definition is expected to hold
-         all data that might be of relevance for future questions.
+         it especially challenging is when the application definition is expected to
+         hold all data that might be of relevance for future questions.
+         
+         We are skeptical if there is one such representation that can fulfill all these
+         aims and interest, while remaining at the same time approachable and executable
+         by a large number of scientists in a community. However, we are also convinced
+         that this is not a reason to accept the status quo of having a very large set
+         of oftentimes strongly overlapping and redundant schemas.
          
-         We are skeptical if there is one representation that can fulfill all these
-         aims, while remaining at the same time approachable and executable by a large
-         number of scientists in a community. With this application definition we 
-         would like to motivate the community to work towards such aim. While doing
-         so we found that existent terminology can be encoded into a more controlled
-         vocabulary.
+         NXem is our proposal to motivate the EM community to work towards more
+         standardization and discussion of what constitutes data, i.e. metadata,
+         numerical and categorical data in research with electron microscopes. We found
+         that existent terminology can be encoded into a more controlled vocabulary.
          
          We have concluded that despite all these details of current EM research with
          SEM, TEM, and focused-ion beam instruments, there a clearly identifiable
          common components and generalizable settings of EM research use cases.
+         Therefore, 
          
          **This application definition has the following components at the top-level:**
          
-         * Generic experimental details (timestamp, identifiers, name);  
+         * Generic experimental details (time-zone-aware timestamp, identifiers, name);  
            conceptually these are session details. A session at a microscope may  
            involve the characterization of multiple specimens. For each specimen  
            an instance of an (NXentry) is created. Details of the instrument have to  
```

Comment:
Redundant with above (multiple specimens with multiple NXentry was already described)

Reply:
edited according to your suggestion